### DISCUSSION TITLE
Please Help!! How to Use GET ??

### DISCUSSION THREAD

**Discussion by brightchip** [Akryum/vue-apollo#1095] [Q&A]
The queries are working with POST by default.
But I need to change it into GET . And I couldn't figure out

`
 apollo: {  
   
    clientConfigs: {
        default: {
            httpEndpoint: process.env.apolloServer,
            persisting: true,
            useGETForHashedQueries: true,
            useGETForQueries: true,
            fetchOptions: {
                useGETForQueries: true
            }
        },
`

**Comment by Akryum** [comment]
> But I need to change it into GET

Why?

**Comment by brightchip** [comment]
Simply because CDN like Cloudflare support GET request for cache rather than POST

But I have found a solution
Create a plugin like below
`
  import { createPersistedQueryLink } from "apollo-link-persisted-queries";
  import { createHttpLink } from "apollo-link-http";
  import { InMemoryCache } from "apollo-cache-inmemory";
  import { ApolloLink } from "apollo-link";


  export default function createApolloClient(context) {
    // Create your custom client...
    const cache = new InMemoryCache();
    const link = ApolloLink.from([
        createPersistedQueryLink({ useGETForHashedQueries: true }),
        createHttpLink({ uri: process.env.apolloServer })
    ]);

    return {
        link: link,
        cache,
        defaultHttpLink: false, // <--- ADD THIS
    };
 } 
``

And change the nuxt.config with
`    
apollo: {
        clientConfigs: {
            default: '~/plugins/_apollo.js',`

**Comment by dmitry** [comment]
Duplicated issue: https://github.com/vuejs/vue-apollo/issues/1091

**Comment by douglasg14b** [comment]
This is actually a pretty important use case when you need to take advantage of CDN caching.
